README.md

```diff
@@ -24,10 +29,51 @@ require('@newrelic/mysql')
 - [`mysql`](https://www.npmjs.com/package/mysql)
 - [`mysql2`](https://www.npmjs.com/package/mysql2)
 
-For more information, please see the agent [installation guide][3], and
-[compatibility and requirements][4].
+For more information, please see the agent [installation guide][3] and the agent [compatibility and
+requirements][4].
+
+Our [API and developer documentation](http://newrelic.github.io/node-newrelic/docs/) for writing
+instrumentation will be of help. We particularly recommend the tutorials and various "shim" API
+documentation.
+
+## Testing
+
+The module includes a suite of functional tests which should be used to verify that your changes
+don't break existing functionality.
+
+These tests rely on a Docker container being setup via scripts in the [Node.js
+agent](https://github.com/newrelic/node-newrelic). This container can be setup by running `npm run
+services` in the main agent repo.
+
+All tests are stored in `tests/` and are written using [Tap](https://www.npmjs.com/package/tap) with
+the extension `.tap.js`.
+
+To run the full suite, run: `npm test`.
+
+## Support
+
+New Relic hosts and moderates an online forum where you can interact with New Relic employees as
+well as other customers to get help and share best practices. Like all official New Relic open
+source projects, there's a related Community topic in the New Relic Explorers Hub. You can find this
+project's topic/threads here: https://discuss.newrelic.com/c/support-products-agents/node-js-agent/.
```

Comment:
Can be follow-up PR... but that's gonna be a broken link. Would use the new tags link.

Reply:
Also looks like we have a lot less info here than usual. I suppose that's fine but I think having the helpful instrumentation documentation links at least would be of benefit. I'd have probably stuck mostly with the template.